UITests/TestBase.cs

```diff
@@ -0,0 +1,50 @@
+﻿//******************************************************************************
+//
+// Copyright (c) 2023 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+using OpenQA.Selenium.Interactions;
+using System.Threading;
+
+namespace UITests
+{
+	public class TestBase
+	{
+		public static WindowsDriver<WindowsElement> Session => SessionManager.Session;
+
+		public static void OpenControlPage(string name)
+		{
+			var search = Session.FindElementByName("Search");
+			search.Clear();
+			Thread.Sleep(1_000);
+			search.SendKeys(name);
+			Thread.Sleep(1_000);
+			Session.FindElementByName(name).Click();
+			Thread.Sleep(5_000);
```

Comment:
Can we do something better than thread sleeps here? This seems unreliable.

E.g. can we wait for a certain piece of UI to update instead?

Reply:
I'm not really sure how you would do that with selenium, if you have any pointers I would love to do that.

Reply:
I also thought about that API, but its quite sophisticated. IIRC, part of that infra was also moved into a package right? Or was that something else.

Regarding the polling approach, I'm not sure if there is something sensible we want to wait for. The idea was to wait for the app to navigate to the page but I don't think we can "poll" for this without introducing UIA elements.

Reply:
In the case of a page navigate you could wait until a known element on the page is available in the UIA tree.